.github/workflows/node-integrate.yml

```diff
@@ -18,6 +18,8 @@ jobs:
     outputs:
       pathChangedAwsLambdaOtelExtension: ${{ steps.pathChanges.outputs.awsLambdaOtelExtension }}
       pathChangedAwsLambdaSdk: ${{ steps.pathChanges.outputs.awsLambdaSdk }}
+      pathChangedSdkSchema: ${{ steps.pathChanges.outputs.sdkSchema}}
+      pathChangedProto: ${{ steps.pathChanges.outputs.proto }}
```

Comment:
@medikoo So I thought that way at first too, but the problem is, if we update the `proto` folder we need to build the new output `ts` that then gets built into the package. 

The actual contents of the `sdk-schema` folder will only change when we bump the version.

In order to run our unit and eventual integration tests against `sdk-schema` we need to be sure they are running against any changes to the `proto` files. Otherwise we may find that a change to the `proto` files broke our tests but we won't know until we actually attempt to cut a release.

Reply:
Ok, indeed, but in such case we need to also ensure that this workflow runs on changes in `proto` (currently it's configured to run only if there are changes in `node`).

Also proto linting feels as runtime-agnostic validation, which could be pursued in dedicated `proto` workflow. still indeed it's valuable to first validate proto before running SDK tests, so it's good we lint here.

Reply:
Good point. I forgot the workflow directory change.